README.md

```diff
@@ -1,2 +1,56 @@
-# Pokedex
-this is a demo of a pokedex proyect
+<div align="center">
+    <h1>POKEDEX</h1>
+        <img src="https://res.cloudinary.com/dvxtbmch0/image/upload/v1644636956/pokebola_s8c3yd.png" alt="Logo" width="150" height="150" />
+</div>
+<details>
+    <summary>Table of Contents</summary>
+    <ul>
+        <li><a href="#about">About</li>       
+        <li><a href="#build-with">Built With</a></li>
+        <li><a href="#installation">Installation</a></li>
+        <li><a href="#usage">Usage</a></li>
+        <li><a href="#contribution">Contributing</a></li>
+        <li><a href="#references">References</a></li>
+    </ul>
+</details>
+<br>
+
+<div id="about" >
+    <h2 align="justify"> Here's a Pokedex powered by PokeAPI build with React.js, that provides information, stats, movements, evolution chain and fuctions like search or suffle to learn about a random Pokémon.</h2>
+</div>
+
+<br>
+<a align="center"> <img src="https://res.cloudinary.com/dvxtbmch0/image/upload/v1644551448/Captura_de_pantalla_de_2022-02-10_21-50-18_va8kpv.png"  width="1000" /> </a>
+
```

Comment:
also we need to add a reference like: this design is made by {author} and we get it from {link}